assert_test.go

```diff
@@ -0,0 +1,55 @@
+package gosnowflake
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+)
+
+func assertNil(t *testing.T, actual interface{}, descriptions ...string) {
+	if actual != nil {
+		desc := joinDescriptions(descriptions...)
+		errMsg := fmt.Sprintf("expected \"%s\" to be nil but was not%s", actual, desc)
```

Comment:
Maybe it would be better to just finish the message "but was not. %s", and just return `strings.Join(descriptions)`?

Reply:
I changed it, but kept method joinDescriptions extracted as a separate method